terratorch/models/backbones/prithvi_vit.py

```diff
@@ -139,13 +141,22 @@ def create_prithvi_vit_100(
         "norm_layer": partial(nn.LayerNorm, eps=1e-6),
         "num_frames": 1,
     }
+
+    # It is possible to overwrite default parameters using 
+    # config file
+    kwargs_ = defaultdict()
+    kwargs_.update(model_args)
+    kwargs_.update(kwargs)
+    kwargs_ = dict(kwargs_)
+
```

Comment:
why is this necessary? the previous line 147 should already cause `model_args` to be overwritten if the same key exists in kwargs. 

```python
>>> a = {"b": 2}
>>> kwargs = {"b": 3, "c": 4}
>>> dict(a, **kwargs)
{'b': 3, 'c': 4}
```

Reply:
That's true. I don't know why it wasn't working for my test.